locale/de/LC_MESSAGES/quest.po

```diff
@@ -18,329 +18,338 @@ msgstr ""
+#: questGen.py:151 questGen.py:168 questGen.py:233 questGen.py:251
+#: questGen.py:308 questGen.py:327
 msgid "or "
 msgstr "oder "
 
-#: questGen.py:141
+#: questGen.py:156
 msgid " with weather boost"
 msgstr " mit Wetterverstärkung"
 
-#: questGen.py:143
+#: questGen.py:158
 msgid " different species of"
 msgstr " verschiedene"
 
-#: questGen.py:156
+#: questGen.py:171
 #, python-brace-format
 msgid "Catch {0} {poke}"
 msgstr "Fange {0} {poke}"
 
-#: questGen.py:158
+#: questGen.py:173
 #, python-brace-format
 msgid "Spin {0} Pokestops or Gyms"
 msgstr "Drehe {0} Pokestops oder Arenen"
 
-#: questGen.py:160
-#, fuzzy, python-brace-format
+#: questGen.py:175
+#, python-brace-format
 msgid "Spin {0} Pokestops you haven't visited before"
 msgstr "Drehe {0} Pokestops die Du noch nicht besucht hast"
 
-#: questGen.py:162
+#: questGen.py:177
 #, python-brace-format
 msgid "Hatch {0} Eggs"
 msgstr "Brüte {0} Eier aus"
 
-#: questGen.py:165
+#: questGen.py:180
 #, python-brace-format
 msgid "Win {0} Gym Battles"
 msgstr "Gewinne {0} Arenakämpfe"
 
-#: questGen.py:167
+#: questGen.py:182
 #, python-brace-format
 msgid "Use a supereffective Charged Attack in {0} Gym battles"
 msgstr "Verwende eine sehr effektive Ladeattacke in {0} Arenakämpfen"
 
-#: questGen.py:169
+#: questGen.py:184
 #, python-brace-format
 msgid "Battle in a Gym {0} times"
 msgstr "Kämpfe {0} mal in einer Arena"
 
-#: questGen.py:172
+#: questGen.py:187
 #, python-brace-format
 msgid "Win {0} Raids"
 msgstr "Gewinne {0} Raids"
 
-#: questGen.py:174
+#: questGen.py:189
 msgid "Win a level 3 or higher raid"
 msgstr "Gewinne einen Level 3 Raid oder höher"
 
-#: questGen.py:176
+#: questGen.py:191
 msgid "Win a level 2 or higher raid"
 msgstr "Gewinne einen Level 2 Raid oder höher"
 
-#: questGen.py:178
+#: questGen.py:193
 #, python-brace-format
 msgid "Battle in {0} Raids"
 msgstr "Kämpfe in {0} Raids"
 
-#: questGen.py:180
+#: questGen.py:195
 #, python-brace-format
 msgid "Transfer {0} Pokemon"
 msgstr "Verschicke {0} Pokemon"
 
-#: questGen.py:182
+#: questGen.py:197
 #, python-brace-format
 msgid "Favourite {0} Pokemon"
 msgstr "Favorisiere {0} Pokemon"
 
-#: questGen.py:184
+#: questGen.py:199
 #, python-brace-format
 msgid "Use {0} {type}Berries to help catch Pokemon"
 msgstr "Verwende {0} {type} Beeren um ein Pokemon zu fangen"
 
-#: questGen.py:189
+#: questGen.py:204
 msgid " Berry"
 msgstr " Beeren"
 
-#: questGen.py:191
+#: questGen.py:206
 #, python-brace-format
 msgid "Power up Pokemon {0} times"
 msgstr "Power up Pokemon {0} mal"
 
-#: questGen.py:193
+#: questGen.py:208
 #, python-brace-format
 msgid "Evolve {0} Pokemon"
 msgstr "Entwickle {0} Pokemon"
 
-#: questGen.py:195
+#: questGen.py:211
 msgid "Use an item to evolve a Pokemon"
 msgstr "Verwende ein Item um ein Pokemon zu entwickeln"
 
-#: questGen.py:197
+#: questGen.py:216
+#, python-brace-format
+msgid "Use a {item} to evolve a Pokemon"
+msgstr "Benutze einem {item}, um ein Pokemon zu entwickeln"
```

Comment:
Instead of "Benutze einem {item}, um ein Pokemon zu entwickeln",
you should use "Benutze {item}, um ein Pokemon zu entwickeln".

Reply:
it's translated to that because that's how it was written in-game - are you sure the sentence works better without `einem`?  (which I now I should be `einen`

![image](https://user-images.githubusercontent.com/5710881/74224599-72636a00-4cb9-11ea-967d-db27c2bb1c13.png)

Reply:
If it's `stein`, use `einen`.

Reply:
Yeah, because of the different evolution items which could be used here. Not all would read correctly, some would be wrong. So leaving it out should work for all of them without loosing valuable information.

Reply:
I see, I will update and remove `ein`